README.md

```diff
@@ -70,14 +75,12 @@ $ source .zshrc
 
 ## Build
 
-Here some instructions how I’m doing things right now:
-* build labs-jdk17
 ```
 # Build Labs OpenJDK17
 $ cd $REPO_DIR
 $ git clone https://github.com/graalvm/labs-openjdk-17
 $ cd labs-openjdk-17
-$ bash configure --with-debug-level=slowdebug --with-boot-jdk=$(echo /path/to/bootjdk/from/above)
```

Comment:
`slowdebug` is really slow. If anything, you might wanna `fastdebug` which enables a bunch of runtime assertions while still using `-O2`. But the default (`release`) should be fine.